modules/creating-an-infra-node.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="creating-an-infra-node_{context}"]
+= Creating an infra node
+
+[IMPORTANT]
+====
+This procedure applies to user-provisioned infrastructure environments only.
+
+See Creating infrastructure machine sets for installer-provisioned infrastructure
+environments or for any cluster where the master nodes are managed by the
+machine API.
+====
+
+Requirements of the cluster dictate that infra node(s) be provisioned. The
```

Comment:
Shouldn't use (s).

Also, I don't understand this first sentence. What requirements are dictating that infra nodes need to be provisioned? I honestly can't even take a stab at an alternative because I don't know what it's supposed to mean.